### PR TITLE
Prepared blog post

### PR DESCRIPTION
**Pull request by cosydney**
This is a case study about prepared 911 after my interview with Amirata

image

### CODE REVIEW

File: data/p/how-axolo-saves-lives.mdx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+draft: false
+summary: "Amirata, a senior software engineer at Prepared911, shared his experience with Axolo in a recent conversation. His insights give an interesting insight into how Axolo has transformed their software development workflow."
+image: "/blog/static/images/prepared/Prepared.jpg"
+authors: ["sydney"]
+---
+
+<ImageContainer
+  alt="Amirata"
+  src="/blog/static/images/prepared/Amirata.jpg"
+  classNameDiv="mx-10 lg:mx-20"
```
Comment (acoudouy) [inline data/p/how-axolo-saves-lives.mdx:14]:
```suggestion
  classNameDiv="mx-10 lg:mx-20 px-0"
```

File: data/p/how-axolo-saves-lives.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+Axolo is a Slack-integrated tool that streamlines code review by creating dedicated Slack channels for each GitHub or GitLab pull request. It enhances team communication, expedites the merging process, and helps manage notifications effectively.
+
+With Axolo, Amirata no longer has to chase after his colleagues for reviews or replies to comments. It's all routed directly to Slack, where he spends most of his work time.
+
+<Callout
+  title="I think for me, the problem Axolo solved is the notification issue. I mean reviews are a big part of my daily communications. I want to either write code or review code and I want to discuss the code."
+  subtitle="You want to write a comment on the exact part of the code in GitHub, but for the rest, it just makes a lot of sense for Slack to facilitate these discussions. That's the problem Axolo solved for me"
+/>
+Amirata
```
Comment (acoudouy) [inline data/p/how-axolo-saves-lives.mdx:68]:
```suggestion
```

File: data/p/how-axolo-saves-lives.mdx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+## Axolo: a Game-Changer
+
+Axolo is a Slack-integrated tool that streamlines code review by creating dedicated Slack channels for each GitHub or GitLab pull request. It enhances team communication, expedites the merging process, and helps manage notifications effectively.
+
+With Axolo, Amirata no longer has to chase after his colleagues for reviews or replies to comments. It's all routed directly to Slack, where he spends most of his work time.
+
+<Callout
+  title="I think for me, the problem Axolo solved is the notification issue. I mean reviews are a big part of my daily communications. I want to either write code or review code and I want to discuss the code."
+  subtitle="You want to write a comment on the exact part of the code in GitHub, but for the rest, it just makes a lot of sense for Slack to facilitate these discussions. That's the problem Axolo solved for me"
```
Comment (acoudouy) [inline data/p/how-axolo-saves-lives.mdx:66]:
```suggestion
  subtitle="You want to write a comment on the exact part of the code in GitHub, but for the rest, it just makes a lot of sense for Slack to facilitate these discussions. That's the problem Axolo solved for me - Amirata"
```

**Comment by cosydney** [comment]
merci

**Pull request merged**